codegen/lib/templates/napi/method_forward.erb

```diff
@@ -62,17 +62,17 @@
     memcpy(returnInstance, &result, sizeof(struct TW<%= entity.name %>));
 
     return External<TW<%= entity.name %>>::New(env, returnInstance, [](Env _env, void* data) {
-        free((TW<%= entity.name %> *) data);
+        free(reinterpret_cast<TW<%= entity.name %> *>(data));
```

Comment:
This is C++ so let's use `new` and `delete` instead of `malloc` and `free`

Reply:
Done